person.rb

```diff
@@ -0,0 +1,25 @@
+# person.rb
+
+class Person
```

Comment:
- [ ] Kindly add this method as well from the project requirement here in the Person class

![image](https://github.com/redmarverick/school_library_people/assets/63473584/24b3c6d5-3436-41f9-8351-57317f385300)